pkg/kubelet/cm/cgroup_manager_unsupported.go

```diff
@@ -38,8 +38,8 @@ func (m *unsupportedCgroupManager) Name(_ CgroupName) string {
 	return ""
 }
 
-func (m *unsupportedCgroupManager) Exists(_ CgroupName) bool {
-	return false
+func (m *unsupportedCgroupManager) Exists(_ CgroupName) error {
+	return nil
```

Comment:
Shouldn't this be an error, to match the behaviour in `pkg/kubelet/cm/cgroup_manager_linux.go`? Also, if the cgroup is unsuported, shouldn't this function always error out?

Reply:
I have changed this to return error.